Casks/u/ua-midi-control.rb

```diff
@@ -0,0 +1,28 @@
+cask "ua-midi-control" do
+  version "5.11.1"
+  sha256 "ced41730d57ebd60b2a5d11426645f7fdadbc218161527c091671e567ce2df56"
+
+  url "https://objects-us-east-1.dream.io/ua-midi-control/builds/UA%20Midi%20Control%20#{version}.zip",
+      verified: "dream.io/ua-midi-control/"
+  name "UA Midi Control"
+  desc "Control-mapping tool for Universal Audio's UAD Console"
+  homepage "https://fonoflow.com/products/ua-midi-control"
+
+  livecheck do
+    url "https://objects-us-east-1.dream.io/ua-midi-control/appcast.xml"
+    strategy :sparkle, &:short_version
+  end
+
+  depends_on macos: ">= :el_capitan"
```

Comment:
```suggestion
  auto_updates true
  depends_on macos: ">= :el_capitan"
```

Based on the present of Sparkle, I assume the app has an in-app updater?

Reply:
I think so, I found the sparkle URL in the package plist. Will add!